use dependencies provided by EasyBuild
authors: Denis Kristak (INUITS), Kenneth Hoste (HPC-UGent)
updated to 24.10.07-2: J. Sassmannshausen (Imperial College London/UK)
diff --git a/r-inla-Version_24.10.07-2.orig/gmrflib/old-stuff/smtp-taucs.c b/r-inla-Version_24.10.07-2/gmrflib/old-stuff/smtp-taucs.c
index 07d3b13..d2964c6 100644
--- a/r-inla-Version_24.10.07-2.orig/gmrflib/old-stuff/smtp-taucs.c
+++ b/r-inla-Version_24.10.07-2/gmrflib/old-stuff/smtp-taucs.c
@@ -43,7 +43,7 @@
 
 #include "GMRFLib/GMRFLib.h"
 #include "GMRFLib/GMRFLibP.h"
-#include "amd.h"
+#include "suitesparse/amd.h"
 #include "metis.h"
 
 GMRFLib_taucs_cache_tp *GMRFLib_taucs_cache_duplicate(GMRFLib_taucs_cache_tp *cache)
diff --git a/r-inla-Version_24.10.07-2.orig/gmrflib/smtp-taucs.c b/r-inla-Version_24.10.07-2/gmrflib/smtp-taucs.c
index ba7331f..dc09aba 100644
--- a/r-inla-Version_24.10.07-2.orig/gmrflib/smtp-taucs.c
+++ b/r-inla-Version_24.10.07-2/gmrflib/smtp-taucs.c
@@ -43,7 +43,7 @@
 
 #include "GMRFLib/GMRFLib.h"
 #include "GMRFLib/GMRFLibP.h"
-#include "amd.h"
+#include "suitesparse/amd.h"
 #include "metis.h"
 
 #define TIME_FUNCTIONS 0
diff --git a/r-inla-Version_24.10.07-2.orig/inlaprog/Makefile b/r-inla-Version_24.10.07-2/inlaprog/Makefile
index c2a5bc3..64e9ede 100644
--- a/r-inla-Version_24.10.07-2.orig/inlaprog/Makefile
+++ b/r-inla-Version_24.10.07-2/inlaprog/Makefile
@@ -2,8 +2,8 @@
 
 PREFIX = /usr/local
 
-RLIB_INC = -DINLA_WITH_LIBR -I/usr/include/R -I/usr/share/R/include
-RLIB_LIB = -L/usr/lib -lRmath -L/usr/lib/R/lib -lR
+RLIB_INC = -DINLA_WITH_LIBR 
+RLIB_LIB = -lRmath -lR
 
 # select compilers and optimized compiler-options, add -fopenmp or similar to use OpenMP
 CC = gcc
@@ -19,7 +19,7 @@ LEXTPREFIX = $(PREFIX)
 
 # The external libraries to link with
 EXTLIBS1 = -L$(PREFIX)/lib -lGMRFLib -L$(LEXTPREFIX)/lib 
-EXTLIBS2 = -lgsl -ltaucs -lmetis -llapack -lblas -lgslcblas -lamd -lmuparser -lz -lgfortran -lcrypto
+EXTLIBS2 = -lgsl -ltaucs -lmetis -lscalapack -lflexiblas -lgslcblas -lamd -lmuparser -lz -lgfortran -lcrypto
 EXTLIBS3 = -Wl,--whole-archive -lpthread -Wl,--no-whole-archive -lm
 
 ################################################################################
@@ -55,8 +55,8 @@ install: $(INLA)
 	 done; \
 	 install -m755 $(INLA) $(PREFIX)/bin/$(INLA);\
 	 install -m755 $(INLA) $(PREFIX)/bin/$(INLA);\
-	 rsync -auv --no-p --no-o --no-g --chmod=ugo=rwX --delete doc/* $(PREFIX)/doc/inla; \
-	 rsync -auv --no-p --no-o --no-g --chmod=ugo=rwX --delete examples/* $(PREFIX)/doc/inla/examples; 
+	 # rsync -auv --no-p --no-o --no-g --chmod=ugo=rwX --delete doc/* $(PREFIX)/doc/inla; \
+	 # rsync -auv --no-p --no-o --no-g --chmod=ugo=rwX --delete examples/* $(PREFIX)/doc/inla/examples; 
 
 clean:; -$(RM) $(OBJ) $(INLA)
 
diff --git a/r-inla-Version_24.10.07-2.orig/inlaprog/src/eval.c b/r-inla-Version_24.10.07-2/inlaprog/src/eval.c
index bc9819b..c352fca 100644
--- a/r-inla-Version_24.10.07-2.orig/inlaprog/src/eval.c
+++ b/r-inla-Version_24.10.07-2/inlaprog/src/eval.c
@@ -35,7 +35,7 @@
 #include <stdio.h>
 
 #if defined(INLA_WITH_MUPARSER)
-#include <muParser/muParserDLL.h>
+#include <muParserDLL.h>
 #endif
 
 #include "GMRFLib/GMRFLib.h"
